mitosheet/installer/mitoinstaller/installer_steps/test_pypi_mitosheet_installer_steps.py

```diff
@@ -22,17 +22,17 @@ def install_step_test_pypi_mitosheet3_check_dependencies():
     raise Exception('Installed JupyterLab: {jupyterlab_version}, Installed extensions {extension_names}'.format(jupyterlab_version=jupyterlab_version, extension_names=extension_names))
 
 
-def install_step_test_pypi_mitosheet3_install_mitosheet3():
-    install_pip_packages('mitosheet3', test_pypi=True)
+def install_step_test_pypi_mitosheet_install_mitosheet3():
```

Comment:
do we want to update the name of this function to .....mitosheet instead of .....mitosheet3? since its installing the mitosheet package